vhdbuilder/packer/install-dependencies.sh

```diff
@@ -262,6 +263,13 @@ for p in ${packages[*]}; do
   capture_benchmark "download_${name}"
 done
 
+downloadContainerdWasmShims
```

Comment:
what is this change ? missing a rebase ?

Reply:
No, in the future WasmShims is one of the packages that will be downloaded via oras, so the install needs to happen after oras is already installed. Moved so that I could verify that this change doesn't hurt anything

Reply:
^ my line of thinking here is wrong. Oras will get installed during build so when it is actually needed during provisioning, it is already installed. So the order does not actually matter - since it will be installed and ready to use during provisioning